CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+Thank you for your interest in helping us to make OpenFastTrace (OFT) better!
+
+# Goal of This Document
+This document aims at answering all the questions potential contributers to OFT might have before feeling ready to get startet.
+
+# What This Document is not
+If you are looking for general information about what OpenFastTrace is, please make sure to check out the [ReadMe](./README.md) that comes with the project.
+
+# What Kind of Contributions can I Make?
+
+## Contributing Code
+If you are a programmer and want to help us improve the implementation, test cases or design of OFT please create a branch of the current `develop` branch of OFT using [git](https://git-scm.com/) and make your changes on that branch.
+
+Then please create a pull request and ask for a review by one of the core team members (e.g. `redcatbear` or `kaklakariada`).
+The reviewers will either ask you to work in review findings or if there are none, merge your branch.
+
+For code contributions we kindly ask you to either pick or create a GitHub [issue in the OFT project](https://github.com/itsallcode/openfasttrace/issues) first which serves context in which the code commit lives.
```

Comment:
Is it necessary to create an issue first? We should make it as simple as possible for contributors.